### ISSUE TITLE
Mismatch between documentation and API

### ISSUE THREAD

**Issue by alexgarel** [pauldmccarthy/indexed_gzip#9]
First of all a big thank you for this project ! 
In the documentation, section [Index Import/Export](https://github.com/pauldmccarthy/indexed_gzip#index-importexport) we have:

```python
fobj = igzip.IndexedGzipFile('big_file.gz')
fobj.build_full_index()
fobj.export_index('big_file.gzidx')
```
but the `IndexedGzipFile` does not have the `build_full_index` and `export_index` methods, they are only available on `_IndexedGzipFile`

So either, the API should evolve to get those functions on `IndexedGzipFile`, either the doc should be:
```python
fobj = igzip.IndexedGzipFile('big_file.gz')
fobj.raw.build_full_index()
fobj.raw.export_index('big_file.gzidx')
```
(notice the use of `raw` attribute which contains the `_IndexedGzipFile` object).

**Comment by pauldmccarthy** [comment]
Hey @alexgarel, thanks for the report! This is an oversight on my part - those methods should definitely be available on the `IndexedGzipFile` class. I'll try and fix this today.

**Comment by pauldmccarthy** [comment]
I have fixed this in the master branch, but am having trouble with travis-ci. I'll keep this issue open until I can get a new version (0.8.3) released.

**Comment by pauldmccarthy** [comment]
This should be fixed in 0.8.4 - thanks, and let me know if you spot anything else!
